### DISCUSSION TITLE
Shell don't work

### DISCUSSION THREAD

**Discussion by tlzfighting** [contiki-ng/contiki-ng#2355] [Q&A]
I add  the module in RPL border router's Makefile.
And I compile the firmware, flash a node, and connect to its serial interface. 
Press enter ,then I don't  get a prompt.Shell don't work?
![图片](https://user-images.githubusercontent.com/95647449/209186125-1102ddcd-cac1-4980-bfc5-616e736e2324.png)

![图片](https://user-images.githubusercontent.com/95647449/209186782-c896c304-aeec-41f6-aee4-1ac9fe65cb80.png)

How can I solve this problem, look forward to your valuable suggestions!

**Comment by CodeDragon82** [comment]
Did you solve this?

**Reply by tlzfighting** [reply]
I solved the problem by replacing the board after
